sympy/utilities/misc.py

```diff
@@ -246,6 +247,10 @@ def debug(*args):
         print(*args, file=sys.stderr)
 
 
+@deprecated(
+    useinstead="the builtin ``shutil.which`` function",
+    issue=19634,
```

Comment:
I assume this number should be changed to match the deprecation issue?

Reply:
Ideally yes, but I think it's fine. The problem is you have to open the deprecation issue to get the number which means you open it before the PR gets pushed.

Ideally the deprecation remove issue would give a succinct explanation of the reason for the deprecation and how to fix code that depends on it with links to the PR where it was removed. If there was a planned timeframe for removal then that could also be mentioned there.

This particular issue seems trivial enough not to need any of these things. I doubt many people are using this function and there is a dropin replacement in the stdlib. What more needs to be said?

There's also no real rush to remove it since the deprecated function isn't doing any particular harm sitting there unused.